src/DragDropAction.vala

```diff
@@ -208,7 +230,13 @@ namespace Gala
 
 							var source_list = sources.@get (drag_id);
 							if (source_list != null) {
+								var dest_list = destinations[drag_id];
 								foreach (var actor in source_list) {
+									// Do not unset reactivity on destinations
```

Comment:
"reactivity"?

Reply:
Reactivity => means that the actor catches input events such as key presses and mouse enters / leaves. I can change that comment if you want.

Reply:
Hm, I didn't actually see that the actual property is called `reactive`, so I guess you're comment is actually really appropriate. Seems weird to explain this reactive term in this location.